lib/routes/mixi2/user.ts

```diff
@@ -0,0 +1,88 @@
+import { type Data, type Route, ViewType } from '@/types';
+import type { Context } from 'hono';
+import { CONFIG_OPTIONS, getClient, parsePost } from './utils';
+import { parseDate } from '@/utils/parse-date';
+
+const handler = async (ctx: Context) => {
+    const limit = Number.parseInt(ctx.req.query('limit') ?? '20', 10);
+    const idOrName = ctx.req.param('id');
+    const mediaOnly = ctx.req.param('media') === 'media';
+
+    let personaId: string;
+    let personaName: string;
+    let personaAvatar: string;
+
+    const client = getClient();
+
+    if (idOrName.startsWith('@')) {
+        personaName = idOrName.slice(1);
+        const userInfo = await client.getPersonaByName({
+            name: idOrName.slice(1),
+        });
+
+        personaId = userInfo.persona.personaId;
+        personaAvatar = userInfo.persona.avatarUrl;
+    } else {
+        const personasData = await client.getPersonas({
+            personaIds: [idOrName],
+        });
+
+        personaId = idOrName;
+        personaName = personasData.personas[0].name;
+        personaAvatar = personasData.personas[0].avatarUrl;
+    }
+
+    const data = await client.getPersonalTimeline({
+        personaId,
+        limit,
+        mediaOnly,
+    });
+
+    return {
+        title: `${personaName} - ${mediaOnly ? 'メディア' : 'ポスト'}`,
+        image: personaAvatar,
+        item:
+            data?.posts
+                ?.filter((post) => !post.isDeleted)
+                .map((post) => ({
+                    title: `@${personaName}`,
+                    description: parsePost(post),
+                    pubDate: parseDate(post.createdAt.seconds * 1e3),
+                    guid: post.postId,
+                })) ?? [],
```

Comment:
This will fail on private profile as `data` only provides 
```json
{
  "postId": "guid",
  "timeSeriesId": "timestamp in ns",
  "visibility": 2
}
```